### ISSUE TITLE
Clase Usuario

### ISSUE THREAD

**Issue by MicaLucas** [PabloSoligo2014/T10#7]
Hola profe, ya subi las modificaciones de la clase, pasando a propiedades los gets y sets. Lo que no entendimos bien es como hacer el atributo de usuario activo, Deberia hacer un Public usuario y que muestre el login ? Algo asi? : Dentro de la clase Usuario
Public Usuario(string description,string login)
{
this.description="Usuario Activo";
this.login=login;
}

**Comment by PabloSoligo2014** [comment]
No, hacer una propiedad booleana Activo, una propiedad como cualquier otra.
